README.md

```diff
@@ -31,7 +31,7 @@ The basic idea is that the build process results in a `bundle.js` file containin
 
 
 #### Code of conduct
-Please be civil when discussing contributions to the Cay front end code and the Coral Project. If in doubt, please consult our [Code of Conduct](https://the-coral-project.gitbooks.io/coral-bible/content/codeofconduct.html)
+Please be civil when discussing contributions to the Cay front end code and the Coral Project. If in doubt, please consult our [Code of Conduct](https://github.com/coralproject/reef/blob/master/CODE-OF-CONDUCT.md).
```

Comment:
I've been taking care to update Codes across the various locations as it evolves, coralproject.net and github primarily. Which is a detail I should add to gitbook when I return. (Or Sydette is welcome to add, cc'd here)



--Typed on my phone

----------------------------
Andrew Losowsky
Website 
Twitter 
LinkedIn 



> On Jan 7, 2016, at 8:39 PM, Riley Davis <notifications@github.com> wrote:
> 
> In README.md:
> 
> > @@ -31,7 +31,7 @@ The basic idea is that the build process results in a `bundle.js` file containin
> >  
> >  
> >  #### Code of conduct
> > -Please be civil when discussing contributions to the Cay front end code and the Coral Project. If in doubt, please consult our [Code of Conduct](https://the-coral-project.gitbooks.io/coral-bible/content/codeofconduct.html)
> > +Please be civil when discussing contributions to the Cay front end code and the Coral Project. If in doubt, please consult our [Code of Conduct](https://github.com/coralproject/reef/blob/master/CODE-OF-CONDUCT.md).
> Is there a "canonical" version? I guess we should point this link at whatever gets updated the most frequently. I thought the gitbooks page was the top level CoC from which all the others originate. @losowsky @jde thoughts?
> 
> —
> Reply to this email directly or view it on GitHub.
>